chapters/ch55.md

```diff
@@ -1,78 +1,82 @@
 > # 55. General case of error attribution
 
---> _replace THIS LINE by your translation for the above line_
+# 55. Trường hợp tổng quát của việc quy lỗi
 
 > Here are the general steps for error attribution. Suppose the pipeline has three steps A, B and C, where A feeds directly into B, and B feeds directly into C.
 
---> _replace THIS LINE by your translation for the above line_
+Đây là những bước tổng quát cho việc quy lỗi. Giả sử một thiết kế pipeline có ba thành phần  A, B, C trong đó A cung cấp thông tin trực tiếp cho B, và B cung cấp thông tin trực tiếp cho C.
 
 ![img](../imgs/C55_01.png)
 
 > For each mistake the system makes on the dev set:
 
---> _replace THIS LINE by your translation for the above line_
+Với từng lỗi gây ra khi hệ thống chạy trên tập phát triển, 
 
 > 1. Try manually modifying A’s output to be a "perfect" output (e.g., the "perfect" bounding box for the cat), and run the rest of the pipeline B, C on this output. If the algorithm now gives a correct output, then this shows that, if only A had given a better output, the overall algorithm’s output would have been correct; thus, you can attribute this error to component A. Otherwise, go on to Step 2.
 
---> _replace THIS LINE by your translation for the above line_
+1. Thử điều chỉnh thủ công kết quả đầu ra ở A cho "hoàn hảo" (ví dụ, chỉnh sửa khung chứa hình mèo cho "hoàn hảo"), và sau đó tiến hành chạy thuật toán cho pipeline gồm có B và C với đầu ra "hoàn hảo" của A đã thao tác thủ công ở trên. Nếu thuật toán cho ra kết quả đầu ra cuối cùng chính xác, điều đó cho ta thấy rằng, thuật toán sẽ cho ra kết quả chính xác khi và chỉ khi việc xử lý thông tin ở công đoạn A cho kết quả đầu ra tốt. Vậy ta có thể quy lỗi cho A. Ngược lại, ta sẽ kiểm chứng thêm ở bước 2. 
```

Comment:
"(ví dụ, ~chỉnh sửa~ _một_ khung chứa hình mèo ~cho~ "hoàn hảo")" -> dư động từ so với bản gốc.

"cho ra kết quả đầu ra cuối cùng" chỗ này hơi bị dư -> "trả về kết quả cuối cùng"

Reply:
"khi và chỉ khi việc xử lý thông tin ở công đoạn A cho kết quả đầu ra tốt" -> "nếu mà A trả về kết quả tốt hơn"
câu này mình góp ý cho sát nghĩa bản gốc hơn.

if and only if == khi và chỉ khi
if only == nếu mà